devicetypes/smartthings/zwave-basic-smoke-alarm.src/zwave-basic-smoke-alarm.groovy

```diff
@@ -173,6 +177,18 @@ def zwaveEvent(physicalgraph.zwave.commands.batteryv1.BatteryReport cmd, results
 	results << createEvent(map)
 }
 
+def zwaveEvent(physicalgraph.zwave.commands.securityv1.SecurityMessageEncapsulation cmd, results) {
+	def encapsulatedCommand = cmd.encapsulatedCommand([ 0x80: 1, 0x84: 1, 0x71: 2, 0x72: 1 ])
+	state.sec = 1
```

Comment:
I've noticed a use of state.sec as 1 and as true. Would setting to a boolean make more sense for consistency?

Reply:
It looks like it's using 1 and 0 pretty much everywhere. I fixed one place in this PR that was using true.

Reply:
Perfect!! Works for me. :)